site/index.rst

```diff
@@ -142,75 +142,78 @@ Getting Started
 .. toctree::
     :maxdepth: 1
    
-    gettingset
+    getting-set/getting-set
   
 
 Course Notes
 ============
 .. toctree::
     :maxdepth: 1
+    :numbered:
    
-    topic1
-    topic2
-    topic3
-    topic4
-    topic5
-    topic5b
-    topic6
-    topic7
-    topic8
-    topic8b
-    topic9
-    topic10
-    topic11
-    topic11b
-    topic12
-    topic13
-    topic13b
-    topic14
-    topic15
-    topic16  
-    topic17
-    topic18
-    topic19
+    topics/intro/intro
```

Comment:
Minor, does rst support "index" files? Like when you navigate to `whatever.com`, you actually go to `whatever.com/index.html`. Would be a nice cleanup here to only say `topics/intro`.

Reply:
I had tried this. Sadly, no dice.

Reply:
actually --- hold on, I have one last idea.

Reply:
the new idea failed too